plugins/package-managers/gradle-inspector/src/funTest/kotlin/GradleAndroidFunTest.kt

```diff
@@ -68,8 +67,13 @@ class GradleAndroidFunTest : StringSpec({
         val definitionFile = getAssetFile("projects/synthetic/gradle-android-cyclic/app/build.gradle").toGradle()
         val expectedResultFile = getAssetFile("projects/synthetic/gradle-android-cyclic-expected-output-app.yml")
 
-        val result = GradleInspectorFactory.create(javaVersion = "17")
-            .resolveDependencies(USER_DIR, listOf(definitionFile), Excludes.EMPTY, AnalyzerConfiguration(), emptyMap())
+        val result = GradleInspectorFactory.create(javaVersion = "17").resolveDependencies(
```

Comment:
Also here `resolveSingleProject()` ? ..and below.

Reply:
> I just noticed the limitation of `resolveSingleProject()` to return just a single `ProjectAnalyzerResult`.

That's exactly the reason why I didn't do a migration here yet. I agree it's probably meaningful to do, but I'd like to consider this out of scope for this PR.